app/services/suggester/users/recent.rb

```diff
@@ -23,7 +23,7 @@ def suggest
       def tagged_article_user_ids(num_weeks = 1)
         Article.published
           .tagged_with(user.decorate.cached_followed_tag_names, any: true)
-          .where("score > ? AND published_at > ?", article_reaction_count, num_weeks.weeks.ago)
+          .where("score >= ? AND published_at > ?", article_score_average, num_weeks.weeks.ago)
```

Comment:
I am not sure why this method was `article_reaction_count` since we are using it to filter by score so I updated it. Probably leftover from a previous code change.